proteus/MeshTools.py

```diff
@@ -1626,8 +1642,8 @@ def convertFromPUMI(self, PUMIMesh, faceList,regList, parallel=False, dim=3):
            self.elementBoundaryNumbering_subdomain2global,
            self.edgeOffsets_subdomain_owned,
            self.edgeNumbering_subdomain2global) = (
-              flcbdfWrappers.convertPUMIPartitionToPython(self.cmesh,
-                  self.subdomainMesh.cmesh))
+               cpartitioning.convertPUMIPartitionToPython(self.cmesh,
```

Comment:
`convertPUMIPartitionToPython` is currently not defined in `cpartitioning` or anywhere else. It used to be in `flcbdfWrappersModule.h`. I can look into reimplementing the function which assigns python object handles that Proteus uses to the cmesh data structures created from an apf/PUMI mesh. Based on syntax, it doesn't look like `cpartitioning` is the right file to put this function anymore. I'll likely move it to `MeshAdaptPUMI.pyx`.